back/src/modules/products/dto/product-characteristics.input.ts

```diff
@@ -24,4 +24,13 @@ export class ProductCharacteristicsInput {
   @MaxLength(100)
   @Field(() => String, { nullable: true })
   two: string;
+
+  /*@IsOptional()
+  @IsNotEmpty()
+  @IsString()
+  @MinLength(2)
+  @MaxLength(100)
+  @Field(() => String, { nullable: true })
+  three: string;*/
```

Comment:
**Suggestion:** Uncomment the code block for the 'three' field. It's not good practice to leave commented code in production, especially if it's functional.
```suggestion
  @IsOptional()
  @IsNotEmpty()
  @IsString()
  @MinLength(2)
  @MaxLength(100)
  @Field(() => String, { nullable: true })
  three: string;
```